raiden/network/transport/matrix/rtc/aio_queue.py

```diff
@@ -17,46 +18,33 @@ def make_wrapped_greenlet(target: Callable, *args: Any, **kwargs: Any) -> Future
     return wrapped_glet
 
 
-class AGTransceiver:
-    def __init__(self) -> None:
-        self.peer_connections: Dict[Address, RTCPeerConnection] = dict()
-        self.event_to_aio_queue = AGQueue()
-        self.event_to_gevent_queue = AGQueue()
-
-    def send_event_to_aio(self, event: Dict[str, Any]) -> None:
-        self.event_to_aio_queue.put(event)
-
-    async def aget_event(self, timeout):
-        try:
-            event = await self.event_to_aio_queue.aget(timeout)
-        except Empty:
-            return {"type": "timeout"}
-        return event
-
-    async def send_event_to_gevent(self, event) -> None:
-        await self.event_to_gevent_queue.aput(event)
+@dataclass
+class RTCPartner:
+    partner_address: Address
+    pc: RTCPeerConnection
```

Comment:
Could you spell this out our use something like `connection`? `pc` itself doesn't make much sense (at least to me).

Reply:
good idea